charts/pega/charts/hazelcast/values.yaml

```diff
@@ -0,0 +1,29 @@
+---
+image: "YOUR_HAZELCAST_IMAGE:TAG"
+imagePullPolicy: "Always"
+# Enter the number of initial members in hazelcast cluster.
+replicas: 3
+# Setting below to true will deploy the infinity in client server hazelcast mode
+enabled: false
+# Memory and CPU settings for HZ service
+resources:
+  requests:
+    memory: "2Gi"
+    cpu: 2
+  limits:
+    memory: "4Gi"
+    cpu: 2
+# client settings for hazelcast
+client:
+  clusterName: "PRPC"
+# Server side settings for hazelcast
+server:
+  java_opts: "-Dhazelcast.initial.min.cluster.size=3 -Djava.util.logging.config.file=/opt/hazelcast/logging.properties -Dhazelcast.jmx=true -XX:MaxRAMPercentage=80.0"
```

Comment:
Have you considered enabled hazelcast diagnostics logging?  It is kind of verbose but hazelcast often asks for it when debugging issues.  If not enabled by default, we should expose an option to provide enable it easily.

Reply:
Created an issue to expose options related to hazelcast diagnostics.